src/languages/en.ts

```diff
@@ -3546,6 +3560,16 @@ export default {
             chatInAdmins: 'Chat in #admins',
             addPaymentCard: 'Add payment card',
         },
+        rules: {
+            individualExpenseRules: {
+                title: 'Expenses',
+                subtitle: 'Set spend controls and defaults for individual expenses. You can also create rules for categories and tags.',
```

Comment:
Did we want "categories" and "tags" to be links?

Reply:
Yes, these should be links :) I've already fixed that

https://github.com/user-attachments/assets/518f595c-12a6-46c5-ae5f-f094ec425daf

https://github.com/user-attachments/assets/b5917818-b1ba-4769-94c9-651aeca4ec53